js/scripts.js

```diff
@@ -0,0 +1,11 @@
+function drawTree(a) {
```

Comment:
give for this argument better name :) names should show to dev on first look what is the method, what it will do and how